falcon_multiqc/commands/save.py

```diff
@@ -148,8 +181,33 @@ def cli(directory, sample_metadata, batch_description, cohort_description, batch
             for line in batch_metadata:
                 split = line.split(",")
                 batch_name = split[0]
+                number_of_samples = split[1] # to be implmented (have to modify models)
```

Comment:
I think remove all these. I didn't put these in purposely because there is no point in specifically saving these to the database, is there?